drivers/include/net/netdev.h

```diff
@@ -54,6 +54,7 @@ extern "C" {
 #include <sys/uio.h>
 
 #include "net/netstats.h"
+#include "net/l2filter.h"
```

Comment:
I'm also against `#ifdef` for header includes (unless 100% needed). Just bloats the top part of the code with more lines and the compiler ignores non-implemented header definitions anyway.